src/wallet/feebumper.cpp

```diff
@@ -246,7 +246,7 @@ bool CFeeBumper::commit(CWallet *pWallet)
         currentResult = BumpFeeResult::MISC_ERROR;
         return false;
     }
-    CWalletTx& oldWtx = pWallet->mapWallet[txid];
+    CWalletTx& oldWtx = pWallet->mapWallet.at(txid);
```

Comment:
Replace with `.find(txid)` above (before `.count()`) and test if exists, also avoids 2nd lookup.